charts/kcp/templates/etcd.yaml

```
@@ -55,13 +54,12 @@ spec:
   secretName: etcd-cert
   duration: 8760h0m0s # 365d
   renewBefore: 360h0m0s # 15d
+  {{ with .Values.certificates.subject }}
```

```suggestion
  {{- with .Values.certificates.subject }}
```
Without the dash, an extra line is added above the `subject:` line.